pkg/strategies/cbpf_playground.go

```diff
@@ -0,0 +1,65 @@
+package strategies
+
+import (
+	. "buzzer/pkg/cbpf/cbpf"
+	"buzzer/pkg/units/units"
+	cpb "buzzer/proto/cbpf_go_proto"
+	fpb "buzzer/proto/ffi_go_proto"
+	pb "buzzer/proto/program_go_proto"
+	"fmt"
+)
+
+func NewCbpfPlaygroundStrategy() *Cbpf_Playground {
+	return &Cbpf_Playground{isFinished: false}
+}
+
+type Cbpf_Playground struct {
+	isFinished bool
+}
+
+func CbpfInstructionSequence(instructions ...*cpb.Instruction) ([]*cpb.Instruction, error) {
```

Comment:
Do we need this? can we just build an array of instructions down in GenerateProgram?

The InstructionSequence function was used for other reasons in the past. Nowadays is just a glorified array builder function :)

Reply:
Ok! I see, we 100% do not need it. Ill remove it